### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Web Server-Certificates: "Esc" key is not working on "actions" button & "Allow Override" button

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#130]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Authorization and press enter
6) Verify on pressing "Esc" key user is able to collapse the Actions button and Allow Override button

**Actual Result:**
User is unable to collapse the "Actions" button and "Allow Override" button menu items using "Esc" key

**Expected Result:**
User should be able to collapse the Actions button and Allow Override button menu items using Esc key 

**User Impact:** 
Screen-Reader users will have difficulty  if Esc key does not behave as it is suppose to and does not closes the menu item

**Attachments:**
[Bug_130_User is unable to collapse the Actions button using ESC key.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327338/Bug_130_User.is.unable.to.collapse.the.Actions.button.using.ESC.key.pptx)

**Comment by jhkimnew** [comment]
The Actions button is a toggle button and users can collapse the control with clicking the button again.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yUsable; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Edge; #C+E-H; #.NETWebsites; #Chrome
